packages/aws-cdk/THIRD_PARTY_LICENSES

```diff
@@ -3241,6 +3241,16 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 
+----------------
```

Comment:
I want Eli to have a look at this.

Reply:
I'm not sure why, but looks like the `aws-sdk` upgrade in the CLI package broke the hoisting.

Currently on master we have:

```console
<repoRoot>/node_modules/aws-sdk (version 2.1094.0)
```

When I upgraded `aws-sdk@^2.979.0` to `aws-sdk@^2.1093.0` and ran `yarn install`, I got the following directory structure:

```console
<repoRoot>/node_modules/aws-sdk (version 2.1094.0)
<repoRoot>/packages/aws-cdk/node_modules/aws-sdk (version 2.1096.0)
```

I expected version 2.1096.0 to be hoisted and replace 2.1094.0 - but thats not what happens. 

So in practice the CLI uses two different versions of `aws-sdk`. One coming from its own direct dependency, which isn't hoisted anymore (2.1096.0), and one coming transitively from `cdk-assets`, which is hoisted (2.1094.0).

We need to figure out why yarn isn't hoisting it. @rix0rrr Any ideas?

Reply:
No, feels like it should be able to. Wonder if there's a constraint somewhere that's preventing the hoist?